compat/src/index.js

```diff
@@ -18,6 +20,15 @@ options.event = e => {
 	return e.nativeEvent = e;
 };
 
+let oldCatchRender = options.catchRender;
+options.catchRender = (error, component) => {
+	if (oldCatchRender && oldCatchRender(error, component)) {
+		return true;
+	}
+
+	return catchRender(error, component);
```

Comment:
this could probably just be `return oldCatchRender && oldCatchRender(error, component) || catchRender(error, component)`

Reply:
Thx. Saved 3B!